.travis.yml

```diff
@@ -11,6 +11,4 @@ install: composer require --no-update guzzlehttp/guzzle $GUZZLE_VERSION; compose
 script: vendor/bin/phpunit
 
 env:
-  - GUZZLE_VERSION: 4.2
-  - GUZZLE_VERSION: 5.0
-  - GUZZLE_VERSION: 5.2
+  - GUZZLE_VERSION: 6.0
```

Comment:
Guzzle 6 needs php 5.5+, so you will need to remove the earlier `5.4` from this file to have passing tests.